.github/workflows/main.yml

```diff
@@ -34,6 +34,11 @@ jobs:
           TEST: API
           PYTHON: ${{ matrix.python }}
           BUILD_DIRECTORY: ${{ github.workspace }}
+          MKL_NUM_THREADS: 2
```

Comment:
Without this training results are varying considerably depending on the number of cores.

Setting it to 1 would be more sensible, but for some bizarre reason doing so causes lightning_KernelSVC kernel="poly" test to fail consistently with difference in 4th decimal digit. It seams like a bug, but for now I'll leave it as 2 as this is the number of cores github actions vm has